File: Mathematics/~Easy/Calculator/Calculator.py
x  = int(input("\n Enter first number  : "))

y  = int(input("\n Enter second number : "))

print("\n 1. Addition")

print("\n 2. Subtraction")

print("\n 3. Multiplication")

print("\n 4. Division")

ch = int(input("\n Enter operation     : "))

print("\n Result = ")

if ch==1 :
    result = x+y
    print(result)
    
elif ch==2 :
    result = x-y
    print(result)
    
elif ch==3 :
    result = x*y
    print(result)
    
elif ch==4 :
    result = x/y
    print(result)
    
else :
    print("Error!")
    
ch=chr(input())

File: Mathematics/~Easy/Calculator/Calculator.cs
using System;

namespace Calculator
{
	class Calculator
	{
		static void Main(string[] args)
		{
			float a, b, c;
			char ch;

			Console.Write("\n Enter first no.: ");
			a = float.Parse(Console.ReadLine());

			Console.Write("\n Enter second no. : ");
			b = float.Parse(Console.ReadLine());

			Console.Write("\n Input Operation: ");
			ch = char.Parse(Console.ReadLine());

			switch (ch)
			{
				case '+': c = a + b;
				Console.Write("\n " + a + " + " + b + " = " + c);
				break;

				case '-': c = a - b;
				Console.Write("\n " + a + " - " + b + " = " + c);
				break;

				case '*': c = a * b;
				Console.Write("\n " + a + " * " + b + " = " + c);
				break;

				case '/': c = a / b;
				Console.Write("\n " + a + " / " + b + " = " + c);
				break;
				
				default:
				Console.Write("\n Error!");
				break;
			}
			Console.ReadKey();
		}
	}
}